### PR TITLE
I completed functionality to draw current view structure and added ne…

### PR DESCRIPTION
**Pull request by marat-sahakyan-923**
…w btn for drawer

### CODE REVIEW

File: src/components/board/Board.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+// eslint-disable-next-line no-unused-vars
```
Comment (M7Arman) [inline src/components/board/Board.jsx]: ?

File: src/components/labTabsComponent/tabs.module.scss [hunk 2]
```diff
@@ -35,10 +35,12 @@
 }
 
 
+
+
 .tabs_btns_section {
     display: flex;
     justify-content: space-between;
-    padding: 0 50px;
+    padding: 0 50px;
```
Comment (M7Arman) [inline src/components/labTabsComponent/tabs.module.scss:43]: ?

**Review by M7Arman — APPROVED** [review]

File: src/components/labTabsComponent/tabs.module.scss (hunk 2 above)
Comment (marat-sahakyan-923) [reply src/components/labTabsComponent/tabs.module.scss:43]: I tried something

File: src/components/board/Board.jsx (hunk 1 above)
Comment (marat-sahakyan-923) [reply src/components/board/Board.jsx]: I deleted this file

**Pull request merged**